templates/commands/render_test.go

```diff
@@ -198,13 +204,16 @@ steps:
 		wantScratchContents  map[string]string
 		wantTemplateContents map[string]string
 		wantDestContents     map[string]string
+		wantFlagInputs       map[string]string
 		wantStdout           string
 		wantErr              string
 	}{
 		{
 			name: "simple_success",
 			flagInputs: map[string]string{
-				"name_to_greet": "🐈",
+				"name_to_greet":   "name",
```

Comment:
Can we make this `name` a real name ("Alice", "Bob", etc)? That will make its behavior a little more self-explanatory.